security/hosts_multitenancy.adoc

```diff
@@ -0,0 +1,109 @@
+[[security-hosts]]
+= Container Hosts and Multi-tenancy
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-hosts-the-os-matters]]
+== The OS Matters
+
+Containers make it easier for developers to build and promote the application
+and its dependencies as a unit. Containers also make it easy to get the most use
+of your servers by simplifying multi-tenancy deployments. You can easily deploy
+multiple applications on a single host, using the kernel and the docker runtime
+to spin up each individual container.
+
+To take full advantage of this packaging and deployment technology, an
+operations team needs the right environment for running containers. You need an
+operating system (OS) that can secure the host kernel and secure containers from
+each other.
```

Comment:
I wonder if parts of http://rhelblog.redhat.com/2015/07/29/architecting-containers-part-1-user-space-vs-kernel-space/ should be re-used or referenced here?